### PR TITLE
Fix UI/packages / Filter search by repo

### PR DESCRIPTION
**Pull request by svtkachenko**
Test results:
```
% pytest tests/foreman/ui/test_packages.py -k 'test_positive_search_in_multiple_repos'
========================================================= test session starts ==========================================================
platform linux2 -- Python 2.7.13, pytest-3.0.7, py-1.4.34, pluggy-0.4.0
rootdir: /home/qui/code/robottelo, inifile:
plugins: xdist-1.18.1, services-1.2.1, mock-1.6.0, html-1.12.0, cov-2.5.1
collected 6 items 
2017-08-01 19:22:56 - conftest - DEBUG - Deselect of WONTFIX BZs is disabled in settings


tests/foreman/ui/test_packages.py .

========================================================== 5 tests deselected ==========================================================
=============================================== 1 passed, 5 deselected in 128.60 seconds ===============================================
```

### CODE REVIEW

File: robottelo/ui/locators/common.py [hunk 1]
```diff
@@ -144,7 +144,7 @@
     "kt_clear_search": (
         By.XPATH, "//button[contains(@ng-click, 'searchCompleted = false')]"),
     "kt_search_no_results": (
-        By.XPATH, "//span[@data-block='no-search-results-message']"),
+        By.XPATH, "//table//span[@data-block='no-search-results-message']"),
```
Comment (svtkachenko) [inline robottelo/ui/locators/common.py]: There 2 elements with such `@data-block` but only one is visible, the one in the results table.

File: robottelo/ui/packages.py [hunk 2]
```diff
@@ -51,3 +51,13 @@ def check_file_list(self, name, file_list):
...
                 locators['package.content_file'] % package_file)
+
+    def search_in_repo(self, name, repo_name, expecting_results=True):
+        """Search for package and filter results by repository"""
+        self.search(name)
+        self.select_repo(repo_name)
+        # In case we expecting that search should not find any entity
+        if expecting_results is False:
+            return self.wait_until_element(
+                common_locators['kt_search_no_results'])
```
Comment (ldjebran) [inline robottelo/ui/packages.py]: please decrease the timeout to 2 seconds

File: robottelo/ui/packages.py [hunk 3]
```diff
@@ -51,3 +51,13 @@ def check_file_list(self, name, file_list):
         for package_file in file_list:
             self.wait_until_element(
                 locators['package.content_file'] % package_file)
+
+    def search_in_repo(self, name, repo_name, expecting_results=True):
```
Comment (ldjebran) [inline robottelo/ui/packages.py]: This will work if we want to search package in all repositories?

Comment (svtkachenko) [inline robottelo/ui/packages.py]: In that case we should use simple search. Actually, it is possible to pass `All repositories` as `repo_name` but it just has no sense :)

**Pull request merged**